tests/conftest.py

```diff
@@ -281,6 +271,22 @@ def _fc(request, fc_module, new_user=False):
     return fc_module
 
 
+def _port(ip=_LOCALHOST):
+    import random
+    from sirepo import const
+
+    r = range(const.TEST_PORT_START, const.TEST_PORT_END)
+    port = random.choice(r)
+    for _ in r:
+        try:
+            _check_port(ip, port)
```

Comment:
Remove "ip" from _check_port(). It's always _LOCALHOST in tests.

Reply:
[2f06d99](https://github.com/radiasoft/sirepo/pull/4846/commits/2f06d994f5be7e9228fe8afef8319ca0979397c3)